dev-util/spirv-llvm-translator/files/0001-Update-LowerOpenCL-pass-to-handle-new-blocks-represn.patch

```diff
@@ -0,0 +1,1116 @@
+From 9ce0fe02fd6cda5fb29fbb0d5037a1798a810b8a Mon Sep 17 00:00:00 2001
```

Comment:
This is too big for ::gentoo, move it to SRC_URI.

Reply:
then we have to put it on our mirrors because the upstream patch needs a `dos2unix`

Reply:
We don't 'put stuff on our mirrors' for years now. We put them in devspace.

Also, there's `edos2unix` in eutils (I think, unless it was moved). Or you can do a similar hack inline.

Reply:
> We don't 'put stuff on our mirrors' for years now. We put them in devspace.

Thank you for the teaching.

> Also, there's `edos2unix` in eutils (I think, unless it was moved). Or you can do a similar hack inline.

Seems to be there. Will give it a try.